app/views/documents/index/_filters.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= form_tag({}, method: :get, data: { gtm: "filter-documents" }) do %>
+<%= form_tag(documents_path, method: :get, data: { gtm: "filter-documents" }) do %>
```

Comment:
Can you explain in the commit message why this change is necessary? If we're only moving/refactoring the test then we shouldn't really need to touch application code.